manifests/kustomize/cluster-scoped-resources/kustomization.yaml

```diff
@@ -1,28 +1,27 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
+namespace: kubeflow
+
+resources:
+- namespace.yaml
 bases:
 - ../base/application/cluster-scoped
 - ../base/argo/cluster-scoped
 - ../base/pipeline/cluster-scoped
 - ../base/cache-deployer/cluster-scoped
-
-resources:
-  - namespace.yaml
-
-# Used by Kustomize
-configMapGenerator:
-  - name: pipeline-cluster-scoped-install-config
-    env: params.env
-
 vars:
-  - name: NAMESPACE
-    objref:
-      kind: ConfigMap
-      name: pipeline-cluster-scoped-install-config
-      apiVersion: v1
-    fieldref:
-      fieldpath: data.namespace
-
+# NOTE: var name must be unique globally to allow composition of multiple kustomize
+# packages. Therefore, we added prefix `kfp-cluster-scoped-` to distinguish it from
+# others.
+- name: kfp-cluster-scoped-namespace
```

Comment:
wondering whether we just use "kfp-" prefix as it can already solve the concern of global conflict with other components. "kfp-cluster-sopced-namespace" would let me think on whether there is another namespace

Reply:
Agreed, any other suggestions?

I wasn't using `kfp-` because I want to be able to compose cluster-scoped package together with kfp namespace scoped package.
Configuration for a var can only appear once between both packages.